.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,40 +0,0 @@
-<!--- This template is meant to help develop good habits in creating PRs. -->
```

Comment:
This file should not be part of this Pull Request.

Reply:
I think merging from master was the problem. 
You might like to try a `git rebase -i ` and removing the commits you did not make from this branch.